internal/commands/app/create.go

```diff
@@ -316,7 +316,7 @@ func (cmd CommandCreate) handleCreateApp(
 		return err
 	}
 
-	if err := appLocal.Load(); err != nil {
+	if err := appLocal.AppData.LoadData(rootDir); err != nil {
```

Comment:
considering `FindApp` will return` !ok` if the app doesn't exist at the path specified by `LoadApp(path string)`, I think it would be appropriate to return an error